Application/Activities/UpdateAttendance.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Application.Interfaces;
+using Domain.Enitities;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Persistence.Data;
+
+namespace Application.Activities
+{
+    public class UpdateAttendance
+    {
+        public class Command : IRequest<Unit>
+        {
+            public Guid Id { get; set; }
+        }
+        public class Handler : IRequestHandler<Command, Unit>
+        {
+            private readonly DataContext _dataContext;
+            private readonly IUserAccesor _userAccesor;
+
+            public Handler(DataContext dataContext, IUserAccesor userAccesor)
+            {
+                _dataContext = dataContext;
+                _userAccesor = userAccesor;
+            }
+
+            public async Task<Unit> Handle(Command request, CancellationToken cancellationToken)
+            {
+                var activity = await _dataContext.Activities.Include(a => a.Attendees)
+                    .ThenInclude(u => u.AppUser)
+                    .SingleOrDefaultAsync(x => x.Id == request.Id);
+
+                if( activity == null) return Unit.Value; ;
```

Comment:
remove space in the beginning of the if clause and remove the extra semi colon

Reply:
Done.